src/components/appNavigation/actionBar/components/ActionButton.js

```diff
@@ -11,26 +12,48 @@ type ActionButtonProps = {
   action: string,
 }
 
-const ActionButton = ({ styles, action }: ActionButtonProps) => {
-  const { actionIcon, trackClicked, goToExternal } = useActionLink(action)
+const ActionButton = ({ styles, action, size = 48, image, isSocial }: ActionButtonProps) => {
```

Comment:
1. move social list (only icons) here
2. add icon prop. if it's set and exists in the list - display icon otherwise the image

Reply:
1. I moved the socials to the component where they are used so that ActionButton stays more generic
2. icon prop is not necessary as it uses the key for loading a fontello icon (key = icon name). I have renamed the lists in the useActionLink hook to have that better noticable